globalbibletools/lexicon-translation
# 3 explorer for finding words to edit

# What has changed

Created a tree view to search by both Hebrew and Greek words.

# Connected Issues

closes #3 

# QA Steps

1. Execute script: ./scripts/extract-byWord-treeItems.js using input file: UBSHebrewDic-v0.9.1-en.JSON as a parameter.
2. Compare output file: ./scripts/treeViewHebrewWordData.json with input file.
3. Execute script: ./scripts/extract-byWord-treeItems.js using input file: UBSGreekNTDic-v1.0-en.JSON
4. Compare output file: treeViewGreekWordData.json with input file.
5. Run Extension
6. Should see new Explorer tree view called: `FIND BY LEMMA`
7. If the correct workspace folder is not open in the explorer there should be no data in tree view.
8. If no workspace folder is open, then use the `Add a folder` link in the `Explorer` view to select the correct workspace folder.
9. Then there should be 1st level nodes of `Hebrew` and `Greek`
10. Selecting each 1st level node should open up a 2nd level of the alphabet of each language.
11. Selecting a 2nd level node should open a 3rd level with each item displaying the first 2 letters of the lemma
12. Selecting a 3rd level node should list all lemmas starting with the first 2 letters of the parent node.
13. Select a lemma and it should open up the corresponding XML file.

# Post-Deployment
